### ISSUE TITLE
"content" tag should be treated separately

### ISSUE THREAD

**Issue by GrenderG** [w3stling/rssreader#135]
At this moment the `content` tag is being treated as `description`, but it should be treated separately (adding the `content` variable along with `setContent()`, `getContent()` methods, etc).

Many feeds have the entire article there (most of WordPress sites, in fact).

**Comment by w3stling** [comment]
All field names in the `Item` class and the `Channel` class are named after the RSS tag names. Atom tags are mapped to a field that best matches a corresponding RSS tag. Therefore is Atom `content` tag mapped to the `description` field. In this way both RSS feeds and Atom feeds can be treated the same.


```java
var mixedFeed = new RssReader().read(List.of(rssFeedUrl, atomFeedUrl));
```

**Comment by w3stling** [comment]
Fixed by PR #195 
Included release 3.9.0
